src/lib/components/actions.svelte

```diff
@@ -127,8 +128,28 @@
 		window.location.href = `${window.location.pathname}?gist=${gistURL}`;
 	};
 
+	const textEncode = (str) => {
+		if (window.TextEncoder) {
+			return new TextEncoder('utf-8').encode(str);
+		}
+		let utf8 = unescape(encodeURIComponent(str));
+		let result = new Uint8Array(utf8.length);
+		for (let i = 0; i < utf8.length; i++) {
+			result[i] = utf8.charCodeAt(i);
+		}
+		return result;
+	};
+
+	const getKrokiCode = (source) => {
+		const data = textEncode(source);
+		const compressed = pako.deflate(data, { level: 9, to: 'string' });
+		let result = btoa(compressed).replace(/\+/g, '-').replace(/\//g, '_');
```

Comment:
Can we use `js-base64`? Moved away from `btoa` due to some compatibility issues.

Reply:
Sure, no probs - changed in 6ab1b29